content/04.methods.md

```diff
@@ -206,8 +204,8 @@ For each project, we then used the `lme4` [@doi:10.18637/jss.v067.i01] R package
 #### Overrepresentation analysis
 
 We next asked whether certain cell types might be overrepresented in one modality compared to the other.
-For this, we first identified cell types of interest as all consensus cell types possible for each project. <!-- TODO: phrasing. -->
-We then created gene sets for each consensus cell type using the project's `CellAssign` marker gene reference.
+For this, we first identified cell types of interest as the set of all possible consensus cell types for each project. <!-- TODO: phrasing. -->
```

Comment:
Just a note that I'm feeling good about where this phrasing ended up, so we could remove this comment or leave it for @jaclyn-taroni's round robin to make sure we get one more set of eyes.